### ISSUE TITLE
GO not restarting after language change (bug)

### ISSUE THREAD

**Issue by paolo2504** [GrandOrgue/grandorgue#795]
I see another post an 'assertion' bug when changing to language other than English.
FOA, let me say I have never experienced this issue.
However, i saw another issue. When changing language , no matter which from-to..., GO asks ",,,WOuld u like to restart Yes/No ?"
Choosing Yes, GO correctly closes itself but does not restart.

**Comment by oleg68** [comment]
I think we shouldn't prompt the user for restarting on windows. Instead, we should put a message box that this setting requires GO to be restarted manually.

**Comment by paolo2504** [comment]
This is an acceptable workaround, even if I don’t understand why it is possibile to do that in Linux and not in Windows. What is the problem ?

**Comment by oleg68** [comment]
Because the same code that works on Linux and Mac does not work on Windows.

**Comment by larspalo** [comment]
I wouldn't mind if we replaced the whole auto restart option on Linux too, into a message that just state that GO must be restarted for the change to take effect. The auto restart does it in the wrong way for me anyway...

**Comment by oleg68** [comment]
@larspalo You have already an option not to restart GO: to press `No` to the prompt.

**Comment by larspalo** [comment]
> You have already an option not to restart GO: to press `No` to the prompt.

Yes, obviously. I'm just saying that the way it's currently done is not necessarily going to work in all cases even under Linux, even though it gives the impression that it should work. And it might make sense to have a uniform behaviour on all platforms...

**Comment by paolo2504** [comment]
Oleg, i do know same code could not work bith in win, linux and mac... btw, i suppose the same source is not compiled for win otherwise it would mean a non-working code is called under WIndows....
Anyway, i believe at this point that this is not considered an issue. Actually, it isn't just let me point out that it would be easy identify the installed OS and run the proper piece of code depending on the returned value.

**Comment by oleg68** [comment]
@paolo2504 Different code for linux and windows is not a problem.

Unfortunally, I could not find any code example for restarting a windows application except very complex solutions with an external program - launcher.
